zkevm-circuits/src/evm_circuit/execution/calldatacopy.rs

```diff
@@ -9,7 +9,10 @@ use crate::{
                 ConstraintBuilder, StepStateTransition,
                 Transition::{Delta, To},
             },
-            memory_gadget::{MemoryAddressGadget, MemoryCopierGasGadget, MemoryExpansionGadget},
+            memory_gadget::{
+                CommonMemoryAddressGadget, MemoryAddressGadget, MemoryCopierGasGadget,
```

Comment:
I think if no this new trait `CommonMemoryAddressGadget` added, don't need to change non oog gadgets like callOp, create etc.

Reply:
It seems that it could not be compiled successfully without `CommonMemoryAddressGadget` import. Since I extract interface functions `offset_rlc`, `length_rlc`, `length` to trait `CommonMemoryAddressGadget`.